src/plugin.ts

```diff
@@ -47,8 +47,8 @@ class AttributeTransform {
       "^\\s*[/\\\\]?" + //Match either slash at the start.
       pathString
         .replace(/^[/\\]?/, "") //Remove any existing leading slash.
-        .replace(/[/\\]/, "[/\\\\]") //Match either slash inside target.
-        .replace(/([$^*+?.()|{}\[\]])/, "\\$1") + //Escape non-slash regex chars.
```

Comment:
Won't square brackets still need to be escaped?

Reply:
I removed them because that line was inadvertently escaping the slash character set brackets in the previous replace. If you want to keep support for escaping them in file paths, I can tweak a bit and push.

Reply:
OK, pushed an update that restores bracket escaping.

Reply:
Oh, I see what you mean. Good pickup. Thanks for adding that back in though, I get a bit OCD about stuff like that 😅